asgs_main.sh

```diff
@@ -2015,7 +2015,7 @@ if [[ $START = coldstart ]]; then
    fi
    # prepare hindcast control (fort.15) file
    # calculate periodic fux data for insertion in fort.15 if necessary
-   if [[ $PERIODICFLUX != null ]]; then
+   if [[ $PERIODICFLUX != 'null' && $FLUXCALCULATOR != 'static' ]]; then
```

Comment:
`FLUXCALCULATOR` is first defined in `config/forcing_defaults.sh` with the value of `static`.